legacy_packages/sdk/test/evm/mock/EntryPoint.json

```diff
@@ -1352,7 +1352,7 @@
bzz-raw://606b2e65d9637db55ac139aea55ea0fe33dda5ba184cd12756678b9c4e30225a\",\"dweb:/ipfs/QmdsQwmA6fNbKKqyKvZnT2jr4aGDqGTbSzJAJpRfsGJPiP\"]},\"contracts/smart-wallet/interfaces/IStakeManager.sol\":{\"keccak256\":\"0x61562da096c3c3da5755bbb5b552ea713723d7be36975cd401022ff321753b73\",\"license\":\"GPL-3.0-only\",\"urls\":[\"bzz-raw://e6bcd63477257007d2fcd92b38c8105cb82c2d0eae3836ac856839234481f4ab\",\"dweb:/ipfs/QmZuNE3y1RvtRXYiw3eiT25YQH2SAa1kFcpxAgW9KX6KtZ\"]},\"contracts/smart-wallet/utils/Entrypoint.sol\":{\"keccak256\":\"0x51d6b65ffde9e43595638b5be8569c2472994d65dea26a3b157addd568c2d89e\",\"license\":\"GPL-3.0\",\"urls\":[\"bzz-raw://388fb384b2ccea051c97dd071fbab0c94ec13284d5e1315147f844a7e3d2a410\",\"dweb:/ipfs/QmUQov9rxwrAFzug9TUQqMWV1Cx1rqtqimWLG5RuSyUHve\"]},\"contracts/smart-wallet/utils/Exec.sol\":{\"keccak256\":\"0x919a01a11adf6d939067ffc3b6bb23de50b3c5ec05e1199a76d1f653d3ff9e7b\",\"license\":\"LGPL-3.0-only\",\"urls\":[\"bzz-raw://2704f818852de962ec11d1f2058d5a1d2491251b3aa384969d011a892d6506dc\",\"dweb:/ipfs/QmWQET87uYdCkExeCZkbc6yikEDmMCBEWzn2Rs8mSjBZ3t\"]},\"contracts/smart-wallet/utils/Helpers.sol\":{\"keccak256\":\"0xb7d1f138c5d0a2d3302c2256d2bd0ab8976baf89edfe0d2de35c5017aeb079c7\",\"license\":\"GPL-3.0\",\"urls\":[\"bzz-raw://a2a8db74e00c5e83f03f35f067a341df50b2171877f1847cd1b544fac6457561\",\"dweb:/ipfs/QmZyhkL5WD97uMmpiXWqjipiWGN1GLhZgadXR56eLQMZEh\"]},\"contracts/smart-wallet/utils/NonceManager.sol\":{\"keccak256\":\"0xba13efc8ff799593affc5217e9ab4aa341316a22bc69eb4e6d53210764bc516b\",\"license\":\"GPL-3.0\",\"urls\":[\"bzz-raw://e82cbabb68c08c5e54379df4b8af4a515193ec84a5fd752de003f99649dabaab\",\"dweb:/ipfs/QmeBkmGAfX1DDM2asBGFUcrmNxYLXWwsX6eEo9tyGPJ5T5\"]},\"contracts/smart-wallet/utils/SenderCreator.sol\":{\"keccak256\":\"0xd5db2a03ce89c3ac228911df32ff2d31aa2aa4d438c7f037b6553cb83c0a8776\",\"license\":\"GPL-3.0\",\"urls\":[\"bzz-raw://d55a923b71c2ffbd406b2e1dafc1ca3514f259b48d89309bbd6772501ff0aa15\",\"dweb:/ipfs/QmQiPjBQxzr96TAPh2ereCsGt4Byii43Tu9gxuZeQP8eVA\"]},\"contracts/smart-wallet/utils/StakeManager.sol\":{\"keccak256\":\"0x8d0a6058f7e86be7eeaa6687ad81500b4672b09e870742acf6b2b937f7e31d26\",\"license\":\"GPL-3.0-only\",\"urls\":[\"bzz-raw://7dab14a949e0c81043299e7ecf12f8fe4395908c6bde8d97066e5b3d86908bfe\",\"dweb:/ipfs/QmdzknZVEZyHKTWrs77yQwPy8pxeUpQ6cpdTPLcH4izB5R\"]},\"contracts/smart-wallet/utils/UserOperation.sol\":{\"keccak256\":\"0x3f3aa133687f459f9fb538835a54aa769418dd6303a02c9fa691690fcc397964\",\"license\":\"GPL-3.0\",\"urls\":[\"bzz-raw://993f8daa2320ca301167767fff33a6989697394fe6dc65905edc4b32368850b5\",\"dweb:/ipfs/Qmaymg8dsE6mrfAWJPs4DXbHR1wxESJnxyFhNkm7AmKBCK\"]},\"lib/openzeppelin-contracts/contracts/security/ReentrancyGuard.sol\":{\"keccak256\":\"0x0e9621f60b2faabe65549f7ed0f24e8853a45c1b7990d47e8160e523683f3935\",\"license\":\"MIT\",\"urls\":[\"bzz-raw://287a2f8d5814dd0f05f22b740f18ca8321acc21c9bd03a6cb2203ea626e2f3f2\",\"dweb:/ipfs/QmZRQv9iuwU817VuqkA2WweiaibKii69x9QxYBBEfbNEud\"]}},\"version\":1}",
```

Comment:
I am not sure why there are changes in this file but this file is generated and should not be updated by hand